swift/common/middleware/s3api/controllers/base.py

```diff
@@ -91,6 +92,21 @@ def resource_type(cls):
         name = cls.__name__[:-len('Controller')]
         return camel_to_snake(name).upper()
 
+    def set_s3api_command(self, req, command):
```

Comment:
Making this a decorator would be nice. (yes I know, there are a few methods where the decorator would not work)